CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+This release fixes an essential bug (#4230) for release health and unhandled/crash events that we introduced with a refactoring (#4101) released in [8.30.0](https://github.com/getsentry/sentry-cocoa/releases/tag/8.30.0).
```

Comment:
```suggestion
This release fixes an essential bug (#4230) for release health and unhandled/crash events that we introduced with a refactoring (#4101) released in [8.30.1](https://github.com/getsentry/sentry-cocoa/releases/tag/8.30.1).
```

Reply:
Ah thanks for the fix. I got the release wrong.